core/spec/models/spree/order_spec.rb

```
@@ -2036,6 +2056,62 @@ def create_adjustment(label, order_or_line_item, amount, source)
+      end
+
+      context 'when payment is void' do
+        let(:amount) { 85 }
+
+        before { order.payments.first.update_column(:state, 'void') }
+
+        it 'returns true' do
+          expect(subject).to be(true)
+        end
+      end
+    end
```

RuboCop (1.81.1)

[convention] Example group has too many memoized helpers [6/5]

(RSpec/MultipleMemoizedHelpers)